apps/client/src/features/rundown/time-input-flow/TimeInputFlow.tsx

```diff
@@ -129,7 +130,7 @@ function TimeInputFlow(props: EventBlockTimerProps) {
       </div>
 
       {warnings.length > 0 && (
-        <div className={style.timerNote}>
+        <div className={style.timerNote} aria-label='warning'>
```

Comment:
I dont think the aria attribute is doing much for us here
Maybe we should reconsider our approach to accessibility, but we shouldnt add code that is not necessary for this PR

Reply:
this was for easier access in the test. or maybe I should use the data-testid, but sometimes when I try that it gets removed by react, not sure when or why

Reply:
yeah, if this is just for tests, data-testid is the right thing